docs/project/changelog.md

```diff
@@ -14,81 +14,67 @@ substitutions:
 
 ## Unreleased
 
-- `pyodide-cdn2.iodide.io` is not available anymore. Please use `https://cdn.jsdelivr.net/pyodide` instead.
+- {{ Enhancement }} Updated Emscripten to version 3.1.27
+  {pr}`2958`, {pr}`2950`, {pr}`3027`, {pr}`3107`, {pr}`3148`, {pr}`3236`,
+  {pr}`3239`, {pr}`3280`, {pr}`3314`
+
+### Project Management
```

Comment:
Maybe
```
## Deployment and testing
```
I have personally never seen "Project Management" as a type of changes in a software changelog, particularly regarding changes that would impact users )

Reply:
Yeah I just had no idea what to call it =)